### PR TITLE
Add either accuracy or micro avg to report

### PR DESCRIPTION
**Pull request by markus-hinsche**
**Status Quo:**
- We use https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html to generate a report of metrics, which includes **either** `accuracy` **or** `micro avg`. 

**Problem**
- When using using Datadog, users expect a metric (like `micro avg` `F1`) to always be reported. Otherwise, they can't visualize trends
- https://github.com/RasaHQ/rasa/issues/8531

**Proposed changes**:
- Always include `micro avg` **and** `accuracy` in `classification_report`

Alternative implementation: would be to re-implement sklearn's [classification_report](https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/metrics/_classification.py#L2003) with slight tweaks

**Checked:**
- Checked that now accuracy is reported when it was previously not ([notebook](https://app.datadoghq.eu/notebook/23688/m-hinsche-jan-26-2022-09-51-investigate-micro-avg-and-accuracy?start=1643180999670&range=14400000&live=true))

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by markus-hinsche** [comment]
/modeltest

```yml
include:
 - dataset: ["financial-demo"]
   config: ["Sparse + BERT + DIET(seq) + ResponseSelector(t2t)"]
```

**Review by carlad — APPROVED** [review]

**Comment by markus-hinsche** [comment]
CC @dakshvar22

File: rasa/model_testing.py [hunk 1]
```diff
@@ -362,9 +363,48 @@ def get_evaluation_metrics(
     f1 = metrics.f1_score(targets, predictions, labels=labels, average="weighted")
     accuracy = metrics.accuracy_score(targets, predictions)
 
+    if output_dict:
+        report = make_classification_report_complete(report, accuracy)
+
     return report, precision, f1, accuracy
 
 
+def make_classification_report_complete(report: dict, accuracy: float) -> dict:
```
Comment (wochinge) [inline rasa/model_testing.py:372]: can we please unit test this?

File: rasa/exceptions.py [hunk 2]
```diff
@@ -67,3 +67,10 @@ def __str__(self) -> Text:
 
 class ActionLimitReached(RasaException):
     """Raised when predicted action limit is reached."""
+
+
+class ClassificationReportException(RasaException):
```
Comment (wochinge) [inline rasa/exceptions.py:72]: `RasaException` means it doesn't get raised to sentry - as this is something which is _not_ supposed to happen it in my opinion should not subclass from `RasaException` as we want it to appear in Sentry, what do you think?

File: tests/test_model_testing.py [hunk 3]
```diff
@@ -128,6 +128,189 @@ def test_get_evaluation_metrics(
...
+                    "recall": 0.5,
+                    "f1-score": 0.6666666666666666,
+                    "support": 2,
+                },
+                "accuracy": 0.8,
+            },
+        ),
+    ],
+)
+def test_make_classification_report_complete(report_in, accuracy, report_out):
```
Comment (wochinge) [inline tests/test_model_testing.py:243]: could you please add type annotations to the test functions?

Thanks in any case for adding the unit tests 🙌🏻

**Review by wochinge — APPROVED** [review]
🚀 Thank you!
